outputs/http.go

```diff
@@ -0,0 +1,66 @@
+package outputs
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/aquasecurity/postee/layout"
+)
+
+type HTTPClient struct {
+	Name    string
+	Client  http.Client
+	URL     *url.URL
+	Method  string
+	Body    string
+	Headers map[string][]string
+}
+
+func (hc *HTTPClient) GetName() string {
+	return hc.Name
+}
+
+func (hc *HTTPClient) Init() error {
+	hc.Name = "HTTP Output"
+	return nil
+}
+
+func (hc HTTPClient) Send(m map[string]string) error {
+	hc.Headers["POSTEE_EVENT"] = []string{m["description"]} // preserve and transmit postee header
```

Comment:
given that output instance is singleton it's better to use variable for headers

Reply:
Could you explain more what you mean by using a variable for header?

Reply:
single instance of class is used. So there is a good chance that `hc.Header` is accessed simultaniosly.
So it will be race condition with unexpected result

Reply:
OK I believe this is what you are alluding to? https://github.com/aquasecurity/postee/pull/249/commits/41d5bf5ba465e56a9f316ac1aaed9e6d16a14a0a